widgets/src/lib/types.ts

```diff
@@ -20,7 +20,9 @@ type Failure = {
 export type Step = {
 	type: RowColumnType;
 	value: string | null;
+	rerun_url: string;
```

Comment:
No all steps are rerunnable:

```suggestion
	rerun_url: string | null;
```